Can't resolve '../build/Release/native_metrics' in /out

I've been using this package for a week or so now, and it's been building fine until just now - both locally and in our build process, I am getting this error when trying to build my project. Can you help?

```
Failed to compile.

./node_modules/applicationinsights-native-metrics/out/main.js
Module not found: Can't resolve '../build/Release/native_metri
cs' in '/Users/***/git/research-center-web/node_modules
/applicationinsights-native-metrics/out'

> Build error occurred
Error: > Build failed because of webpack errors
    at Object.build [as default] (/Users/***/git/research-center-web/node_modules/next/dist/build/index.js:192:15)
    at processTicksAndRejections (internal/process/task_queues.js:86:5)
error Command failed with exit code 1.
```

I'm getting a similar error, even though in my case I only need this package indirectly as a dependency of `vscode-extension-telemetry`.

Here is the error I get:
```
ERROR in ./node_modules/applicationinsights-native-metrics/out/main.js
Module not found: Error: Can't resolve '../build/Release/native_metrics' in 'C:\Users\...\node_modules\applicationinsights-native-metrics\out'
```

I checked and the `native_metrics.node` file does exist locally at the expected path.

This error is related to the use of webpack: `applicationinsights-native-metrics` tries to load `native_metrics.node` through `require("../build/Release/native_metrics")`, but it seems that webpack is unable to load .node files by default.

It is possible to configure webpack to load .node files, using `node-loader` package for example, but:
- `node-loader` doesn't seem to be supported anymore. Alternatives exist (`awesome-node-loader`), but I didn't try them.
- Given that `applicationinsights-native-metrics` uses `require("../build/Release/native_metrics")` without specifying the `.node` extension, it would mean creating a webpack loader rule specifically for this dependency, which isn't great.

@markwolff > Could you please consider adding the `.node` extension explicitely, i.e. `require("../build/Release/native_metrics.node")`?
That way, it would be possible to:
- Add a dependency on node-loader: `npm i --save-dev node-loader`
- Add a loader rule in `webpack.config.js` file:
```
rules: [
    {
        test: /\.node$/,
        loader: 'node-loader'
    }
]
``` 

If anyone has a better alternative, I'm also interested in it. :)

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ os:
   - linux
   - osx
 node_js:
-  - "13"
+  - "14"
   - "12"
   - "10"
   - "8"
@@ -23,7 +23,7 @@ script:
   - npm test
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then docker build --build-arg VERSION=$TRAVIS_NODE_VERSION -t tmp/alpine-native-metrics .; fi;
 after_success:
-  if ([ "$TRAVIS_BRANCH" == "master" ] || [ ! -z "$TRAVIS_TAG" ]) && 
+  if ([ "$TRAVIS_BRANCH" == "master" ] || [ ! -z "$TRAVIS_TAG" ]) &&
       [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
     docker run -e NODE_PRE_GYP_GITHUB_TOKEN tmp/alpine-native-metrics; fi;
     echo "Publishing"; npm install node-pre-gyp-github@1.4.3; ./node_modules/.bin/node-pre-gyp configure; ./node_modules/.bin/node-pre-gyp build; ./node_modules/.bin/node-pre-gyp package; ./node_modules/.bin/node-pre-gyp-github publish;

--- a/lib/main.ts
+++ b/lib/main.ts
@@ -1,5 +1,5 @@
 const events = require("events");
-const natives = require("../build/Release/native_metrics");
+const natives = require("../build/Release/native_metrics.node");
 
 type gcTypes =
   | "Scavenge"

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "applicationinsights-native-metrics",
   "description": "Native APM agent for the Application Insights Node.js SDK",
-  "version": "0.0.5",
+  "version": "0.0.6",
   "engines": {
     "node": ">=6.0.0",
     "npm": ">=3.8.6"

--- a/package.json
+++ b/package.json
@@ -27,8 +27,7 @@
     "test:renamebinding": "rename-files ./ binding.gyp temp",
     "test:renametestbinding": "cp test.gyp binding.gyp",
     "test:renameoriginalbinding": "rename-files ./ temp binding.gyp",
-    "clean": "rimraf build/",
-    "publish:github": "echo 'Publishing'; npm install node-pre-gyp-github@1.4.3; ./node_modules/.bin/node-pre-gyp configure; ./node_modules/.bin/node-pre-gyp build; ./node_modules/.bin/node-pre-gyp package; ./node_modules/.bin/node-pre-gyp-github publish;"
+    "clean": "rimraf build/"
   },
   "gypfile": true,
   "license": "MIT",
@@ -38,7 +37,7 @@
   },
   "dependencies": {
     "nan": "^2.14.0",
-    "node-pre-gyp": "^0.12.0"
+    "@mapbox/node-pre-gyp": "^1.0.0"
   },
   "devDependencies": {
     "@types/node": "^11.15.2",

